src/project.rs

```diff
@@ -10,29 +11,106 @@ use crate::interpreter::ast;
 
 pub const PROJECT_EXTENSION: &str = "hurban";
 
+#[derive(Debug, Clone)]
+pub enum NextAction {
```

Comment:
Let's implement `Copy` here. Clippy should afterwards point out places where this is cloned (there's a few cases in lib.rs)